Source/Core/VideoCommon/GraphicsModSystem/Runtime/Actions/SetSettingsAction.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright 2023 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "VideoCommon/GraphicsModSystem/Runtime/Actions/SetSettingsAction.h"
+
+#include <picojson.h>
```

Comment:
```suggestion
#include "VideoCommon/GraphicsModSystem/Runtime/Actions/SetSettingsAction.h"

#include <memory>

#include <picojson.h>
```
To avoid an indirect inclusion.